### PR TITLE
Update SAC v102.0 install instructions for Linux

### PR DESCRIPTION
**Pull request by wangliang1989**
目前只在ubuntu的云服务器上进行了试验。似乎是因为sac增加了下载数据等功能，造成编译需要新的库文件（lib开头的文件，我不知道是不是该叫库文件）。
这样看来，其他发行版上还必须进行试验。目前已经完成了：

- [x] Ubuntu 20.04.2 LTS by @wangliang1989 
- [x] Centos 7 by @wangliang1989 

如果你们想做哪个，动手前先在这by一下自己，免得重复了，你们不做的话，我会逐渐做的。@core-man @seisman

### CODE REVIEW

**Comment by core-man** [comment]
Just a short comment. See https://docs.github.com/cn/actions/hosting-your-own-runners/about-self-hosted-runners#linux, I guess we could add an installation test on those different machines via GitHub Actions. Ping @seisman @wangliang1989 for the comment.

**Comment by seisman** [comment]
> Just a short comment. See [docs.github.com/cn/actions/hosting-your-own-runners/about-self-hosted-runners#linux](https://docs.github.com/cn/actions/hosting-your-own-runners/about-self-hosted-runners#linux), I guess we could add an installation test on those different machines via GitHub Actions. Ping @seisman @wangliang1989 for the comment.

SAC source code is not publicly available, so we can't use github actions.

**Comment by wangliang1989** [comment]
> > Just a short comment. See [docs.github.com/cn/actions/hosting-your-own-runners/about-self-hosted-runners#linux](https://docs.github.com/cn/actions/hosting-your-own-runners/about-self-hosted-runners#linux), I guess we could add an installation test on those different machines via GitHub Actions. Ping @seisman @wangliang1989 for the comment.
> 
> SAC source code is not publicly available, so we can't use github actions.

建一个私有的repo，把代码传上去

**Comment by wangliang1989** [comment]
@core-man @seisman
用github action？我也可以专门装虚拟机，你们做决定吧

**Comment by seisman** [comment]
We won't use github actions for this purpose.

**Comment by wangliang1989** [comment]
> We won't use github actions for this purpose.

拿我逐渐抽时间安装虚拟机把主流的都试一遍

**Comment by seisman** [comment]
Ubuntu and Debian are the same, Fedora and CentOS are the same, so we only need to test two of them.

File: source/install/linux-install.rst [hunk 1]
```diff
@@ -56,23 +56,30 @@ Linux 下安装 SAC，可以直接安装二进制包，也可以从源码安装
     $ sudo apt install build-essential
     $ sudo apt install libncurses5-dev libsm-dev libice-dev
     $ sudo apt install libxpm-dev libx11-dev zlib1g-dev
+    $ sudo apt install libedit-dev libxml2-dev
+    $ sudo apt install libcurl4-openssl-dev
```
Comment (seisman) [inline source/install/linux-install.rst]: Linux 二进制包不需要安装类似的依赖？

Comment (wangliang1989) [reply source/install/linux-install.rst]: 先试验的二进制包，没有出问题

**Comment by seisman** [comment]
It's a little complicated, because we have merged https://github.com/seisman/SAC_Docs_zh/pull/210 into master, but the same changes are not changed into dev branch.

**Comment by wangliang1989** [comment]
> It's a little complicated, because we have merged #210 into master, but the same changes are not changed into dev branch.

之前我就说过会这样，当时你说出现这个问题，你来处理😄

File: source/install/linux-install.rst (hunk 1 above)
Comment (seisman) [inline source/install/linux-install.rst]:
```suggestion
    $ sudo apt install libedit-dev libxml2-dev libcurl4-openssl-dev
```

**Review by seisman — APPROVED** [review]

**Pull request merged**